### PR TITLE
feat(HAWK-988): Refactor Admin Server Terraform

### PR DESCRIPTION
**Pull request by austinsherron**
* Move admin server terraform into self-contained module
* Add call to new module in root module

### CODE REVIEW

File: gcp/admin-server/main.tf [hunk 1]
```diff
@@ -44,7 +44,6 @@ sudo dnf install -y google-cloud-cli-gke-gcloud-auth-plugin 2>> $log
   metadata          = {}
   resource_policies = []
   tags              = []
-  depends_on        = [google_project_service.compute]
```
Comment (austinsherron) [inline gcp/admin-server/main.tf:47]: I don't think this is strictly necessary.

Comment (mackyi) [reply gcp/admin-server/main.tf:47]: how does this dependency get calculated without this line?

File: gcp/vars.tf [hunk 2]
```diff
@@ -46,6 +46,6 @@ variable "package_url" {
 
 variable "admin_server_instance_type" {
   type        = string
-  description = "URL of the Project N package to install on launch"
-  default = "n1-standard-1"
+  description = "The admin server's instance type"
+  default     = "n2-standard-1"
```
Comment (mackyi) [inline gcp/vars.tf:50]: do we need to specify this in 2 separate places?

Comment (austinsherron) [reply gcp/vars.tf:50]: Yes.

File: gcp/admin-server/main.tf (hunk 1 above)
Comment (austinsherron) [reply gcp/admin-server/main.tf:47]: It's not necessary.

**Review by mackyi — APPROVED** [review]

**Pull request merged**